project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import sbt.Setting
+import scoverage.ScoverageKeys
+
+object CodeCoverageSettings {
+
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    ".*Routes.*",
+    ".*Reverse.*",
+    "app.*",
+    "prod.*",
+    "com.kenshoo.play.metrics.*",
+    "uk.gov.hmrc.incorporatedentityidentification.featureswitch.api.*",
+    "uk.gov.hmrc.incorporatedentityidentification.testonly"
+  )
+
+  val settings: Seq[Setting[_]] = Seq(
+    ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimum := 90,
+    ScoverageKeys.coverageFailOnMinimum := true,
```

Comment:
Do we need to fail on minimum?

Reply:
I'm not certain. A quick search through our repositories revealed a fairly mixed approach with an even distribution of true and false. Personally, I prefer "true" as it forces you to get the test coverage up to the required limit.